database/db_locations.py

```diff
@@ -0,0 +1,100 @@
+from database.db_base import DBBase
+
+
+class DBLocations(DBBase):
+    def __init__(self, database):
+        super().__init__(database)
+
+        self.t_cities = '/cities'
+        self.t_locations = '/locations'
+
+    def get_cities(self):
+        """
+        Return all available cities in database.
+        :return: (dict) {city_id: "city_name", ...}
+        """
+        return self.db.get(self.t_cities, None)
+
+    def get_location_info(self, city_id, location_id):
```

Comment:
Do we need to pass a city_id? 
Looks like the next id is calculated in respect to the whole table here https://github.com/dsc-ucu-lviv/hvalue/blob/48e392e4da676ed104aad8f91eb1e9f905dd5f50/database/db_base.py#L5